mysql-test/suite/rocksdb/t/prefix_extractor_override.test

```diff
@@ -0,0 +1,101 @@
+--source include/have_rocksdb.inc
+
+--disable_warnings
+DROP TABLE IF EXISTS t1;
+--enable_warnings
+
+
+SET @@global.rocksdb_update_cf_options = 'default={disable_auto_compactions=true;};';
```

Comment:
This is required in general but not in this particular case, because the server is restarted further in the testcase